torch/_inductor/codegen/cpp_wrapper_cuda.py

```diff
@@ -78,8 +78,8 @@ def write_header(self):
             do {                                               \\
                 CUresult code = EXPR;                          \\
                 const char *msg;                               \\
-                cuGetErrorString(code, &msg);                  \\
-                if (code != CUDA_SUCCESS) {                    \\
+                CUresult err = cuGetErrorString(code, &msg);                  \\
+                if (err != CUDA_SUCCESS || code != CUDA_SUCCESS) {                    \\
```

Comment:
If `err != CUDA_SUCCESS` can we assume that `msg` is populated?

Reply:
As per the documentation, if `err != CUDA_SUCCESS` then msg will be set to NULL.

See: https://docs.nvidia.com/cuda/cuda-driver-api/group__CUDA__ERROR.html#group__CUDA__ERROR_1g72758fcaf05b5c7fac5c25ead9445ada